### ISSUE TITLE
RKSV §9, Abs. 2, Z 2: "fortlaufende Nummer des Barumsatzes"

### ISSUE THREAD

**Issue by cewaldcompeso** [a-sit-plus/at-registrierkassen-mustercode#371]
Situation: mehrere Kassen an einer Datenbank. Jede Kasse hat ihre Signatureinheit und schreibt ein Datenerfassugnsprotokoll. 
Die fortlaufende Nummer des Barumsatzes für das DEP ist ja gleichzusetzen mit einer Belegnummer. Diese wird von unserem System jedoch zentral eindeutig je Haus (und nicht je Kasse) erstellt. Können wir die Belegnummer als eindeutige Nummer verwenden (Lücken im DEP), oder müssen wir zusätzlich eine Nummerierung je Kasse und DEP implementieren?

**Comment by daJedi** [comment]
selbst wenn eine Signaturverkettung eindeutig gegeben ist?

**Comment by ErichFreitag** [comment]
Ja, weil wenn ich sie richtig verstanden habe dann in jeder Kasse eine Menge Lücken wären. Wegen Verkettung nicht müssen, aber wegen Zusammenhang sollen.

**Comment by daJedi** [comment]
Durch Verkettung ist ja ein Zusammenhang gegeben, aber ich verstehe schon, was du meinst. Danke.

**Comment by aecoraIT** [comment]
Naja, eine Registrierkasse ist lt. RKSV als System definiert, welche über ein eigenes DEP besitzt und signieren kann (überigens auch interessant für die Förderung). Somit muss man sich die einzelne Kasse, und nicht das Gesamtsystem anschauen. Und da ist es schon Nachvollziehbar mit fortlaufenden Belegnummern PRO Kasse zu arbeiten, damit keine Lücken entstehen. Ich würde aber sowieso (auch für die Übersicht) die Belegnummer auf Kassen aufgeteilt vergeben, und diese so in die zentrale Datenbank schreibe z.B. 001/00001 ; 002/00001 usw. und dann jeweils eideutig fortlaufen. Damit ersparen Sie Ihrem Kunden ärger, bei einer Finanzkontrolle, wenn dann Belege zwar richtig verifiziert werden können, aber es augenscheinlich "Lücken" im DEP gibt. Da wird jeder Kontrolleur von Natur aus etwas stutzig ;)

**Comment by peda** [comment]
Vorweg sei gesagt, dass das nur meine Interpretation ist. Aber Lücken würde ich kritisch sehen, da auch mit Sicherheitseinrichtung der jeweils letzte Beleg gelöscht werden kann ohne, dass Spuren hinterlassen werden. Eine große Anzahl an Lücken könnte hier einen Verdachtsmoment darstellen der eventuell eine genauer Prüfung nach sich ziehen könnte. Deshalb würde ich im eigenen Interesse versuchen Lücken so gut es geht zu vermeiden.

**Comment by ep27** [comment]
Dazu möchte ich etwas anmerken:

Wenn ich auf einem System Barrechnungen und Überweisungsrechnungen ausstelle, muss ich ja nur die Barrechnungen in die Registrierkasse eintragen. Ich wüsste jedoch nicht, warum ich da nicht den gleichen Nummernkreis hernehmen sollte. Und insgesamt sind ja keine Lücken in den Rechnungsnummern, sondern eben nur in der Registrierkasse.

Und eine fortlaufende Nummer sagt nichts aus über die Integrität - wenn ich schon die Rechnung aus dem Datenerfassungsprotokoll löschen kann (was dem Benutzer des Systems ohnehin nicht möglich sein sollte) warum sollte dann nicht noch einmal die gleiche Rechnungsnummer vergeben werden können?

**Comment by ErichFreitag** [comment]
Die RKSV-Belegnummer muss wegen der Algorithmen auf Lebenszeit einer Kasse eindeutig sein.

Die hat aber zunächst mit einer Rechnungsnummer nichts zu tun. Darum wie oben "nicht müssen, aber sollen" - sonst entstehen unnötige Aufwände durch genauere Detailprüfungen, warum denn die Barumsätze nicht wie vorgegeben fortlaufend nummeriert sind.

Technisch ist es egal - der Kunde hat irgendwann möglicherweise gröberen Erklärungsbedarf.

**Comment by ep27** [comment]
Es kommt mMn da schon auf den Fall an. Wenn ich keine eigene Kasse für die Barumsätze habe, sondern immer eine Rechnung ausdrucke - nur manchmal eben als Barrechnung ist es wohl nicht sinnvoll, einen eigenen Rechnungskreis für Barrechnungen zu erstellen. Und die Lücken sind ja vom Kunden gut erklärbar - und es ist eben nicht vorgegeben, dass Barumsätze fortlaufend nummeriert werden müssen. Es müssen nur die Rechnungen fortlaufend nummeriert werden und wenn Barrechnungen und Überweisungsrechnungen auftreten, ist das eben so.

Und einen zusätzlichen Belegkreis für die Registrierkasse zu verwenden, der jetzt mit der Rechnung direkt nichts zu tun hat ist wohl keine gute Idee. Und ich glaube, so lange es keine Überschneidungen bei den Belegnummern geben kann ist es sogar erlaubt, dass man zwei vorhandene Rechnungskreise in eine Registrierkasse eintippt.

**Comment by ErichFreitag** [comment]
"Fortlaufende Nummer des Barumsatzes" ist schon ziemlich eindeutig.

**Comment by ep27** [comment]
Wenn man einen Handwerker hernimmt, der zwischen den normalen Rechnungen auch ein paar Barrechnungen dabei hat (und er womöglich auch gar nicht weiß welche Art es werden wird, wenn er mit der Rechnung anfängt), aber trotzdem registrierkassenpflichtig ist, wäre ein zweiter Nummernkreis für die Barrechnungen doch Overkill und würde mehr Probleme machen als lösen. Wenn Lücken sein dürfen, was durch "Fortlaufende Nummer des Barumsatzes" wie ich zugeben muss, nicht wirklich zum Ausdruck kommt, ist es für mich ok.

**Comment by ep27** [comment]
Wenn ich mich dafür entscheide, eine zusätzliche registrierkassen-interne Belegnummer zu verwenden - muss die auch im Klartext auf der Rechnung stehen oder reicht es, wenn sie im Strichcode kodiert ist. Eine zusätzlich angegebene Belegnummer auf der Rechnung wo schon eine Rechnungsnummer vorhanden ist, wäre doch verwirrend.

Dadurch könnte ich die Lücken vermeiden. Zur Not könnte ich natürlich Zahlung Nr. hinschreiben, wenn sich nicht dann ein Prüfer daran stört, dass nicht die Rechnungsnummer als Belegnummer genommen wird.

**Comment by lebail** [comment]
Der Erlass beschreibt im Punkt 4.6 die von der RKSV zusätzlich (im Vergleich zur BAO) anzugebenden Belegdaten. Ein Klartext-Ausdruck der Belegnummer wird weder im Erlass noch in der RKSV selbst gefordert.

Wenn Sie also bereits eine Rechnungsnummer zur Identifikation des Geschäftsfalles haben, interpretiere ich das so, dass Sie die RKSV-Belegnummer _nicht_ im Klartext angeben müssen. Achtung: Das ist nur meine persönliche Meinung.

Erlass, Punkt 4.6:

> Neben den Beleganforderungen des § 132a Abs. 3 BAO (Mindestanforderungen) ist § 132a
> Abs. 8 iVm § 11 RKSV ab 1. April 2017 beachtlich, weil die Belegangaben durch die RKSV
> erweitert werden, um insbesondere die einzelnen Geschäftsvorfälle nachvollziehen zu können
> (siehe § 11 RKSV: Kassenidentifikationsnummer, Uhrzeit der Belegausstellung,
> maschinenlesbarer Code und Betrag der Barzahlung nach Steuersätzen getrennt).

**Comment by ep27** [comment]
Vielen Dank, vielleicht übernehme ich die eigentliche Rechnungsnummer noch als zus. Feld.
